.github/actions/upload-alerts/action.yml

```diff
@@ -19,9 +19,20 @@ runs:
       with:
         repository: pytorch/test-infra
         path: _test-infra
+
     - name: Run Script to Validate Alerts
-      if: github.event_name != 'pull_request'
-      run: python tools/scripts/validate_alerts.py --alerts "${{ inputs.alerts }}"
+      shell: bash
+      # if: github.event_name != 'pull_request'
```

Comment:
Unrelated to this PR but I think you could consider having a dry run mode in these script so that you don't need to comment out `github.event_name != 'pull_request'` to test